installvm.py

```diff
@@ -437,6 +440,10 @@ def createKickstart(self):
 
         partition_string = ''
         multipath_string = ''
+        kernel_params = 'mitigations=auto quiet crashkernel=1024M '
+        if vmParser.args.kernel_params != '':
+            kernel_params=kernel_params+vmParser.args.kernel_params
```

Comment:
please use string way to concatenate string, i know we have other places as well if possible can you please send another fix commit for same

Reply:
changed above, the whole code cleanup can be different commit